packages/moonstone/styles/internal/fonts.less

```diff
@@ -99,13 +100,6 @@
 .buildFontFace(@font-family-base-name; @font-system-src-moonstone-condensed-600; { font-stretch: condensed; font-weight: 600; });
 .buildFontFace(@font-family-base-name; @font-system-src-moonstone-condensed-700; { font-stretch: condensed; font-weight: 700; });
 
-
-/* ----- Manual Moonstone Locale Fonts (Stacking onto "Moonstone") ------ */
```

Comment:
Please do not remove this override.

Reply:
If `ar`, `he` and `th` locales are handled in this language block for applying `LG Smart UI AR HE TH`, this lines are useless.

Reply:
Please refer to the earlier comment on the PR regarding {{AR HE TH}}.